frontend/src/pages/ProjectSettings/ProjectSettings.tsx

```diff
@@ -91,128 +94,133 @@ const ProjectSettings = () => {
 				<title>Project Settings</title>
 			</Helmet>
 
-			<Box>
-				<Box style={{ maxWidth: 560 }} my="40" mx="auto">
-					<Heading mt="16" level="h4">
-						Project Settings
-					</Heading>
-					<div className={styles.tabsContainer}>
-						<ProjectSettingsContextProvider
-							value={{
-								allProjectSettings,
-								setAllProjectSettings,
-								loading,
+			<Box style={{ maxWidth: 560 }} my="40" mx="auto">
+				<Heading mt="16" level="h4">
+					Project Settings
+				</Heading>
+				<div className={styles.tabsContainer}>
```

Comment:
I'm not sure what is going on but the header looks bunch up as a result of these changes:

![Screenshot_2023-08-11_at_3_17_30_PM](https://github.com/highlight/highlight/assets/58678/b2b0340f-f78c-44d6-b394-8c0e6c7152c2)

Compared to what is on prod:


![Screenshot 2023-08-11 at 3 19 59 PM](https://github.com/highlight/highlight/assets/58678/ccc72628-824e-431a-941c-7c9c90d3ac93)

Reply:
Interesting... I am not actually seeing this on render or locally, so curious what you are looking at to repro